### PR TITLE
OAS remove encoding which is breaking forward slash `/` in URL constructions

### PR DESCRIPTION
**Pull request by mkendall07**

### CODE REVIEW

File: ads/openadstream.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ export function openadstream(global, data) {
   checkData(data, ['adhost', 'sitepage', 'pos', 'query']);
   //required fields
   validateDataExists(data, ['adhost', 'sitepage', 'pos']);
-  let url = 'https://' + encodeURIComponent(data.adhost) + '/3/' + encodeURIComponent(data.sitepage) + '/1' + new String(Math.random()).substring(2, 11) + '@' + data.pos;
+  let url = 'https://' + data.adhost + '/3/' + data.sitepage + '/1' + new String(Math.random()).substring(2, 11) + '@' + data.pos;
```
Comment (cramforce) [inline ads/openadstream.js]: Why did you remove it from the `data.host`?

While we are here: Remove the `new String(`. It does nothing.

Comment (jdelhommeau) [inline ads/openadstream.js]: if you remove the new String, you can't make the substring op anymore on the Math.random result. So it's either that or transforming through some other way to string.

The encodeURIComponent is not needed as well for the host, as it can causes same issue than the one for the data.sitepage.

Comment (cramforce) [inline ads/openadstream.js]: What would be the issue on the host?

If you want a string use `String` instead of `new String`. The latter does something subtly different from what it seems to do.

**Comment by mkendall07** [comment]
Updated per comments. Encoding the host is probably not required but it doesn't harm to leave it there.

**Comment by cramforce** [comment]
LGTM

**Comment by mkendall07** [comment]
Thanks. What's the block pending?

**Comment by jridgewell** [comment]
> What's the block pending?

We have a freeze on non-critical changes until tomorrow.

**Comment by mkendall07** [comment]
Good to merge now?
